tools/openms/IsobaricAnalyzer.xml

```diff
@@ -224,7 +224,7 @@
       <option value="tmt6plex">tmt6plex</option>
     </param>
     <param name="param_in" type="data" format="mzml" optional="False" label="input raw/picked data file" help="(-in) "/>
-    <param name="param_id_pool" type="text" size="30" label="ID pool file to DocumentID's for all generated output files" help="(-id_pool) Disabled by default. (Set to 'main' to use /home/thouwaar/Downloads/openms-2.0-Linux-x86_64/share/OpenMS/IDPool/IDPool.txt)">
+    <param name="param_id_pool" type="text" size="30" label="ID pool file to DocumentID's for all generated output files" help="(-id_pool) Disabled by default. (Set to 'main' to use /home/houwaart/Projects/OpenMS2.0/bin_15_07_2015/share/OpenMS/IDPool/IDPool.txt)">
```

Comment:
I blacklisted id_pool. If people need this, we have to find out what this does .. :)